package.json

```diff
@@ -56,6 +58,9 @@
   "dependencies": {
     "dat.gui": "^0.7.7",
     "deepmerge": "^4.2.2",
+    "jsx": "^0.9.89",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
@vim-sroberge was this intentionally placed in this repo? My understanding is that this repo (vim-webgl-viewer) is intended to define the API (without React) whereas the vim-webgl-app repo defines the demo app code (which can contain React).

Reply:
Good catch I need to remove it.